### PR TITLE
touch: update page

### PR DESCRIPTION
**Pull request by jimmychang851129**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by jimmychang851129** [comment]
IMO, former description is lengthy and not intuitive, so I propose a change to the description.

Feel free to let me know if there is any suggestion.

File: pages/common/touch.md [hunk 1]
```diff
@@ -15,6 +15,6 @@
 
 `touch -c -t {{YYYYMMDDHHMM.SS}} {{path/to/file1 path/to/file2 ...}}`
 
-- Set the file time of a specific file to the time of anothe[r] file and don't [c]reate file if it doesn't exist:
+- Set the files' timestamp to the [r]ef_file's timestamp, and don't [c]reate file if not exist:
```
Comment (gutjuri) [inline pages/common/touch.md:18]:
```suggestion
- Set the files' timestamp to the [r]eference file's timestamp, and do not [c]reate the file if it does not exist:
```

I assume that ref_file stands for reference file?

**Review by gutjuri — COMMENTED** [review]
Thanks! I agree with your change, however, I propose to use "reference file" instead of ref_file in the description. Also, I edited the second half-sentence of the description to sound more natural (to me).

**Review by spageektti — APPROVED** [review]
LGTM. Thank you for your contribution!

**Pull request merged**